### ISSUE TITLE
Focus issue on item open

### ISSUE THREAD

**Issue by joe-allen-89** [cgkineo/adapt-hotgrid#109]
### Subject of the issue/enhancement/features
When selecting an item the focus jumps to the arrows/close buttons rather than the content, when you navigate past this to the content you then have to navigate back to the arrows/close buttons. 

### Your environment
* v5.25.2
* Chrome with Jaws 2013 & NVDA

### Steps to reproduce
Navigate to hotgrid using screen reader, open item and navigate around. 

### Expected behaviour
On item open focus should be straight to the content and the user able to move past that to the arrows/close buttons.

### Actual behaviour
Focus jumps to arrows/close buttons, the user then has to navigate past these to get to the content.
